website/src/mode.tsx

```diff
@@ -225,3 +236,43 @@ const TagSetOption = (p: { radio: RadioStateReturn; level: string }) => {
     </Tooltip>
   )
 }
+
+export const PrefPanel = () => {
+  const preferences = useContext(preferencesContext)
+  return (
+    <div>
+      <h3>Display Mode:&ensp; </h3>
+      {<ExperiencePicker onSelect={preferences.expLevelUpdate} />}
```

Comment:
I'm not sure why you have `ensp` and a space here. Also, I don't think you need headings here, normal `label` elements should work just fine. And then instead of ensp you just need a normal space. To write a space between elements on different lines, you can put `{" "}` on the first line to make an explicit space.

Reply:
The label elements look a lot like regular text, even similar to the regular text. I'm not sure if this is really what we want?

![image](https://user-images.githubusercontent.com/92126390/161323615-131eed16-7fc0-404b-ba86-ff695936cb6f.png)